packages/client/components/BaseButton.tsx

```diff
@@ -41,6 +41,7 @@ export interface BaseButtonProps extends PlainButtonProps {
   size?: 'small' | 'medium' | 'large'
   children?: ReactNode
   className?: string
+  buttonName?: string
```

Comment:
nit-- it'd be nice to have a prop name used for naming testing attributes for all components, not just buttons. maybe we just have a `cy` or `dataCy` prop? that way it'll be the same across divs, forms, buttons, etc.

Reply:
Yeah that makes sense to me. I'll make that change now.